spec/fixtures/vcr_cassettes/users.yml

```diff
@@ -107,19 +107,19 @@ http_interactions:
       encoding: UTF-8
       string: <?xml version='1.0' encoding='UTF-8'?><tsResponse xmlns="http://tableau.com/api"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://tableau.com/api
-        http://tableau.com/api/ts-api-3.1.xsd"><user id="07a635e6-fde8-4ca8-ba78-d301fd19fdcb"
+        http://tableau.com/api/ts-api-3.1.xsd"><user id="ce93aaeb-fb25-4fbd-a573-74b549f5e95b"
         name="test" siteRole="ExplorerCanPublish" authSetting="ServerDefault" externalAuthUserId=""/></tsResponse>
     http_version:
-  recorded_at: Wed, 28 Oct 2020 22:10:19 GMT
+  recorded_at: Tue, 24 Nov 2020 22:00:06 GMT
 - request:
     method: get
-    uri: http://TABLEAU_HOST/api/3.1/sites/f31923d2-dea4-4470-843a-9fc999d240aa/users?pageNumber=1&pageSize=100
+    uri: http://TABLEAU_HOST/api/3.1/sites/fb50c166-f809-44e0-995e-2cf56ceffbc0/users?pageNumber=1&pageSize=100
     body:
       encoding: US-ASCII
       string: ''
     headers:
       X-Tableau-Auth:
-      - uRTWEYOxQRi2XsUiy58Wiw|xz3JBXTLpZ8d83WgENB1xTPJ42mua5Hp
```

Comment:
should user agent have changed a few lines down?

Reply:
I didn't regenerate all the cassettes after the user-agent change, just jobs.yml. Should I update everything?